lib/Commands/BuildEngineCommand.cpp

```diff
@@ -90,7 +90,7 @@ struct AckermannKey {
 
   /// Create an Ackermann key from the encoded representation.
   AckermannKey(const core::KeyType& key) {
-      auto keyString = StringRef(key);
+      auto keyString = static_cast<StringRef>(key);
```

Comment:
`key.str()` seems nicer here, for example

Reply:
However... I find this patch unclear to understand the benefit without actually understand how code like this would use the args. I'm a little worried that we will land something like this, then actually go and use the args, and then maybe end up with questions about whether it was a good idea. I think I'd like to see how code like this uses the args before weighing in on this.